admin/addseries.php

```diff
@@ -36,11 +40,24 @@
 						VALUES ('$series_name', '$volume_number')";
 
 						if (mysqli_query ( $connection, $sql )) {
-							echo $series_name . ' Volume '. $volume_number . ' series created in database.';
+							echo '<p>' . $series_name . ' Volume '. $volume_number . ' series created in database.</p>';
 						} else {
-							echo "Error: " . $sql . "<br>" . mysqli_error ( $connection );
-						}
-					}
+							echo "<p>Error: " . $sql . "<br>" . mysqli_error ( $connection ) . '</p>';
+						} ?>
+						<p>Use the form below to add another new series to your database.</p>
+						<form method="post" action="<?php echo $filename; ?>" class="form-inline">
+							<div class="form-group">
+								<label for="series_name">Series Name</label>
+								<input name="series_name" class="form-control" type="text" size="50" value="" required />
+							</div>
+							<div class="form-group">
+								<label for="volume_number">Volume #</label>
+	          		<input name="volume_number" class="form-control" type="text" size="3" maxlength="4" value="" required />
+          		</div>
+							<input type="hidden" name="submitted" value="yes" />
+							<input type="submit" name="submit" value="Submit" class="btn btn-primary form-submit" />
+						</form>
+					<?php }
```

Comment:
Need to clean this up a little, but wanted the option to add another series after adding the first one.